client/src/containers/Economy/components/Compare/components/Table/index.tsx

```diff
@@ -19,6 +20,9 @@ export const Table: FC<TableProps> = observer(({ indicator, unit }) => {
   return (
     <Fetch country={'all'} indicator={indicator}>
       {({ data, isLoading, error }) => {
+        const countriesAll =  JSON.parse(JSON.stringify(countries));
```

Comment:
Don't do stringify and parsing. You already have a JSON structure (object)

Reply:
ok @MaksymRudnyi